src/components/offer-card/OfferCard.tsx

```diff
@@ -34,18 +35,21 @@ const OfferCard: FC<OfferCardProps> = ({
     proficiencyLevel
   } = offer
 
-  const fullName = `${authorFirstName} ${authorLastName}`
+  const fullName = `${authorFirstName} ${authorLastName[0]}.`
```

Comment:
Does lastName now is an array?

Reply:
nope, its a string. I needed to get first character and used bracket notation for it

Reply:
ok, didn`t know that you can do like that